kod/object/passive/spell/jala.kod

```diff
@@ -160,11 +160,22 @@ messages:
          if event = EVENT_DAMAGE
             OR event = EVENT_ATTACK
             OR event = EVENT_RUN
-            OR (event = EVENT_CAST AND Send(Send(SYS, @GetSettings), @CanRecastJalaWithNecklace))
+            OR (event = EVENT_CAST AND Send(Send(SYS, @GetSettings), @CanRecastJalaWithNecklace)
+               AND NOT IsClass(what,&Jala))
          {
             return FALSE;
          }
 
+         if event = EVENT_USE & usewhat <> $
+         {
+            if NOT IsClass(usewhat,&Food)
```

Comment:
These class checks are unexpected.  There should be a comment here explaining why they're here.

Reply:
I added a comment on why food, and instruments, and potions are different then other items that get used.